lib/elements/satin_column.py

```diff
@@ -1052,6 +1056,16 @@ def to_stitch_groups(self, last_patch=None):
         # beziers.  The boundary points between beziers serve as "checkpoints",
         # allowing the user to control how the zigzags flow around corners.
 
+        # If no seed is defined, compute one randomly using time to seed,  otherwise, use stored seed
+
+        if self.use_seed == 0:
+            random.seed()
+            x = random.randint(1, 10000)
+            random.seed(x)
+            self.set_param("use_seed", x)
+        else:
+            random.seed(self.use_seed)
```

Comment:
I would rather not rely on a hidden global like this. I'm working on a PR that uses the stateless interface from `hashlib` instead of `random` and maintains a counter in a local variable.

Reply:
looking forward to see your work . I'm sure I will learn a lot from it... Thanks for that opportunity.

Reply:
I'm looking forward to seeing what both of you come up with for this.  I had envisioned it as an actual user-settable parameter, visible in the Params window, maybe with a randomize button much like how Inkscape's live path effects work.  The UI may be a little tricky though.

Reply:
The hashlib version will help with that since it will allow seeds to be arbitrary strings.